### DISCUSSION TITLE
Android

### DISCUSSION THREAD

**Discussion by KNAGORA** [Micke-K/IntuneManagement#225] [Ideas]
are there other people that would like to see not only windows policies but also android policies/configs/ apps/... ?

**Comment by Micke-K** [comment]
Hello,

I had a look. I can see many of these policies. eg Settings Catalog and Templates.

Any specific policy type you are missing?

It does not have support for Linux Compliance Policies. I missed that Microsoft uses a different compliance policy API for Linux.

Cheers!

**Comment by KNAGORA** [comment]
i miss the android configuration policies. it would be a nice thing to have to.
